src/cbc_sdk/base.py

```diff
@@ -1428,6 +1428,73 @@ def not_(self, q=None, **kwargs):
         return self
 
 
+class CriteriaBuilderSupportMixin:
```

Comment:
This makes the "criteria" support somewhat more complex, but I'm sure the complexity is in a good cause.

Reply:
This is simpler instead of having _update_criteria, update_criteria, and add_criteria in every query implementation that implements the functions the exact same way. Though this I also noticed the difference in behavior between alerts, device, and few other models and processes/events so I made the experience consistent across all models now.